composer.json

```
@@ -16,7 +16,7 @@
     ],
     "require": {
         "php": ">5.4",
-        "paragonie/random_compat": "~1.0|~2.0|9.99.99",
+        "paragonie/random_compat": "~1.0|~2.0|9.99.*",
```

Maybe better to use the `~` prefix to follow the style of the other version constraints?
```suggestion
        "paragonie/random_compat": "~1.0|~2.0|~9.99.99",
```

---

Sure, let's keep things consistent! 👍